server/@types/bapv.d.ts

```diff
@@ -159,12 +159,13 @@ export type BAPVVisitBalances = VisitBalances & {
 // Visit slots, for representing data derived from VisitSessions
 export type VisitSlot = {
   id: string
+  sessionTemplateReference: string
```

Comment:
This can ultimately replace `id` here but will need consideration of existing data that lacks `sessionTemplateReference`.